applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_solver.py

```diff
@@ -76,6 +87,40 @@ def __init__(self, main_model_part, custom_settings):
             "displacement_absolute_tolerance": 1.0e-9,
             "residual_relative_tolerance": 1.0e-4,
             "residual_absolute_tolerance": 1.0e-9,
+            "adaptative_remesh_settings" : {
```

Comment:
I am not sure if this should be in the baseclass of the StructuralMechanics solver
1. It does only support a few elements
2. Does it works for dynamic problems too?
3. I think (please correct me if I understood it wrong) if the element provides some kind of measure as basis for the remeshing then also other Physics (e.g. Fluid) could use this
I am thinking abt sth @msandre showed me some time ago, he did sth very similar if I remember correctly: Using sensitivities as basis for remeshing, which he aso did with mmg
=> Therefore it might make sense to have this in the MeshingApp and integrate it from there in the solvers. 
@armingeiser and @MFusseder are working on the Sensitivities for the Structural, maybe one could also make use of this then

Reply:
It works in dynamics too, in any case I can remove directly, the convergence criteria knows the default settings

Reply:
Thinking that last comment, maybe is not a good idea, because in that case the arguments will be static

Reply:
Certainly I don't know how to solve this problem

Reply:
the big question is why we need it as a part of the base solver. can't this remeshign be done as a process?